src/screens/ParadeMapScreen/LocationCard.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+import React from "react";
+import { Image, StyleSheet, View } from "react-native";
+import ContentPadding from "../../components/ContentPadding";
+import Text, { scaleWithFont } from "../../components/Text";
+import locationCardDefault from "../../../assets/images/locationCardDefault.png";
+import { blackTwentyColor, whiteColor } from "../../constants/colors";
+
+type Props = {
+  name?: string,
+  text?: string,
+  image?: any
```

Comment:
it's not my intention to leave this `any` here, I'm just unsure as to how to type it so it support the (eventual) url string from contentful + the local file source referenced in the `defaultProps`...

Reply:
I would just type this to be the local asset for now. If this is still a problem I recommend using `mixed` instead of `any` (if flow is ok with that). Alternatively, because this is the first time this is being used you could completely omit passing the image and just hard code it in the component for now.